cloudify/plugin_installer.py

```diff
@@ -558,3 +558,16 @@ def get_pth_dir(venv=None):
         return '{0}/lib/python{1}/site-packages'.format(prefix, version)
     else:
         raise NonRecoverableError('Unsupported OS: {0}'.format(os.name))
+
+
+def _python_executable(plugin):
+    if 'supported_py_versions' not in plugin:
+        return sys.executable
+    if 'py310' in plugin['supported_py_versions']:
+        return sys.executable
```

Comment:
Let's also have a check here, that 3.10 is indeed the version we're on (`sys.version_info`)

We'll soon probably add 3.11 here too!